update UI for android launch

johngruen opened the pull request:
@devinreams  This branch is not done yet, but it's where I'm updating the UI, I'll remove the WIP flag and tag you for review when it's good to go.

devinreams commented:
Thanks @johngruen, so far so good 👍 

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/49511/54046851-c228df80-4192-11e9-97f7-1796e96d8dc0.png">

johngruen commented:
@devinreams i think this is ready for review.... a few notes: 
- i've added `footer_links` in addition to `header_links`. These can serve as site navigation on the new homepage.
- i've added links to the app stores for google and app to the `_config.yml`
- i've left in src files for all weights of Inter even through the site only imports three weights

johngruen commented:
@devinreams this is ready for real review

pdehaan commented on _layouts/home.html:
```diff
@@ -0,0 +1,32 @@
+          </div>
+        <div class="home__app-buttons">
+          <a class="app-store" href="{{ site.ios_link | escape }}">
+            <img src="{{relativeBase}}assets/images/app-store.svg" alt="Apple App Store" height=" 50px" width="auto">
```
I don't think the `height=` tag likes "px" suffixes (that's only a CSS thing IIRC). Probably smart enough to ignore it, but...

Probably same w/ `width="auto"`.

Once this is hosted somewhere, I can run Google Lighthouse and aXe and webhint.io scanners though and check for other potential a11y issues.

devinreams commented:
Looks great! I just tested all the pages locally and made super sure the FAQ anchors still jump as expected (no reason they wouldn't just saying I _did_ test).

Quick scan of the code, looks fine (@pdehaan has eagle eyes for this though).

@johngruen want us to go ahead and merge so it's live and then we can run more testing and add any more potential todos to #98?

devinreams commented:
![Screen Shot 2019-03-15 at 10 46 25 AM](https://user-images.githubusercontent.com/49511/54447900-1e01e400-4710-11e9-8555-96d1d97a5b27.png)

![Screen Shot 2019-03-15 at 10 48 26 AM](https://user-images.githubusercontent.com/49511/54447908-21956b00-4710-11e9-8eaf-20be3a193ba9.png)

johngruen commented:
Okay @devinreams i fixed @pdehaan 's issue

pdehaan commented on _layouts/home.html:
```diff
@@ -0,0 +1,32 @@
+        </div>
+        </div>
+        <div class="home__content-right">
+          <img src="./assets/images/devices.png">
```
I built this locally and aXe says:

> Images must have alternate text

![Firefox_Lockbox___Firefox_Lockbox_project_documentation_and_website](https://user-images.githubusercontent.com/557895/54449560-a467f700-470b-11e9-85e5-469c0d705ee2.png)

devinreams replied:
Added alt text, thanks @pdehaan 🙏
